components/core/src/clp_s/CommandLineArguments.cpp

```diff
@@ -287,11 +287,20 @@ CommandLineArguments::parse_arguments(int argc, char const** argv) {
             extraction_options.add(input_options);
 
             po::options_description decompression_options("Decompression Options");
+            // clang-format off
             decompression_options.add_options()(
                     "ordered",
                     po::bool_switch(&m_ordered_decompression),
                     "Enable decompression in ascending timestamp order for this archive"
+            )(
+                    "ordered-chunk-split-threshold",
+                    po::value<size_t>(&m_ordered_chunk_split_threshold)
+                            ->default_value(m_ordered_chunk_split_threshold),
+                    "Number of records to include in each output chunk when decompressing records "
+                    "in order"
```

Comment:
```suggestion
                    "in timestamp ascending order"
```

Reply:
I replaced with "in ascending timestamp order" instead.